packages/components/addon/components/hds/app-footer/status-link.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+export const STATUSES = {
+  OPERATIONAL: {
+    text: 'Operational',
+    iconName: 'check-circle',
+    iconColor: 'var(--token-color-foreground-success)',
+  },
+  DEGRADED: {
+    text: 'Degraded',
+    iconName: 'alert-triangle',
+    iconColor: 'var(--token-color-foreground-warning)',
+  },
+  MAINTENANCE: {
+    text: 'Maintenance',
+    iconName: 'alert-triangle',
+    iconColor: 'var(--token-color-foreground-warning)',
+  },
+  CRITICAL: {
+    text: 'Critical',
+    iconName: 'x-circle',
+    iconColor: 'var(--token-color-foreground-critical)',
+  },
+};
```

Comment:
Curious if these need to be similar in naming, but I have the _StatusLink text as: 
System operation
System degraded
System maintenance 
System outage

Reply:
The "text" is the actual text content so I can update the values, thanks.

Did you mean "System operation**al**" for the first one? I can update the desired text values after you double check, thanks.

Reply:
Updating these...